R/calc_aggregate_counts.R

```diff
@@ -13,8 +13,8 @@
 #' if all_dates is FALSE for all variables EXCEPT .Confirmed and .Deaths
 #' @param ucla_only logical, only consider data from UCLA
 #' @param state logical, return state level data
-#' @param collapse_vaccine logical, combine vaccine variables for more
-#' intuitive comparisons
+#' @param collapse_vaccine logical, add variable for Residents.Initiated, equal to Residents.Completed, 
```

Comment:
What you've done here is super useful, thank you. It's made me realized that I don't think `collapse_vaccine` is an intuitive name for this parameter -- maybe `sub_vax_data` for consistency with the new function you've created? I know that introducing this change would mean we'd need to make other downstream changes where this function is called, so I'm curious to hear your thoughts on that idea.

Reply:
This is a good point. I did a search of the `covid19_behind_bars_scrapers` repo and there are no instances of `calc_aggregate_counts` being called with an argument specified for `collapse_vaccine`, so I will go ahead and change this.